content/docs/guides/crosswalk/kubernetes/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Pulumi Crosswalk for Kubernetes"
+linktitle: Crosswalk for Kubernetes
+menu:
+  userguides:
+    parent: guides
+    identifier: crosswalk-kubernetes
+
+aliases: ["/docs/reference/crosswalk/kubernetes/"]
+---
+
+Pulumi Crosswalk for Kubernetes is a collection of industry standard
+best-practices and tasks for managing Kubernetes and its infrastructure in production.
+
+<center><img src="/images/docs/quickstart/kubernetes/cake.svg" width="670"></center>
+
+In this guide we will show you, using batteries-included examples, how to **provision**
```

Comment:
Nit, remove bold from provision and use?

Reply:
Done